pypesto/storage/read_from_hdf5.py

```diff
@@ -64,9 +65,9 @@ def read(self, objective: Objective = None) -> Problem:
         problem:
             A problem instance with all attributes read in.
         """
+        # create empty problem
         if objective is None:
             objective = Objective()
```

Comment:
```suggestion
            objective = ObjectiveBase()
```
we will need to rewrite this (result should not contain the objective, just a "problem info", so fine for the moment.